### DISCUSSION TITLE
Hosting in Azure Kubernetes and using Azure Active Directory

### DISCUSSION THREAD

**Discussion by R4cOOn** [datalust/seq-tickets#1587] [Help] [answered]
Hi,

I've managed to get Seq working in Azure AKS using Azure Active Directory thanks to https://docs.datalust.co/docs/azure-active-directory
I struggled until I read the instructions more carefully and noticed that I needed to set the `SEQ_API_CANONICALURI ` environment variable.

For now I set it directly in the deployed Helm chart.
I looked at the charts on https://github.com/datalust/helm.datalust.co/tree/main/charts/seq but didn't see any option to override that environment variable through chart value overrides.
What is the expected way to set that environment variable using Helm?

I also noticed on ticket https://github.com/datalust/seq-tickets/discussions/1438 that it is recommended to set `SEQ_FEATURES_ENABLED = nativemetastore` but I can't see this variable in the Helm chart either.
Could you please let me know if it still needs to be set?
And can I set it even though I already have Seq running or will it wipe all the existing data?

Cheers.

**Comment by nblumhardt** [comment, marked as answer]
Hi @R4cOOn, thanks for getting in touch.

The value to set in the helm chart is called `baseUri` - it's listed in _values.yaml_ at:

https://github.com/datalust/helm.datalust.co/blob/main/charts/seq/values.yaml#L20

The `nativemetastore` option is no longer required or supported as of Seq 2022 - it's now the default.

Hope this helps!
Nick

**Reply by R4cOOn** [reply]
Thanks for the quick reply.
Everything is working fine with the Helm update.

Cheers.
